t/settest_notnull_and_prettyprint/settest_notnull_and_prettyprint.def

```diff
@@ -0,0 +1,17 @@
+
+openDate          = 09/03/2014 at 10:34am EDT
+dueDate           = 09/10/2014 at 10:34am EDT
+answerDate        = 09/10/2014 at 10:34am EDT
+paperHeaderFile   = defaultHeader
+screenHeaderFile  = defaultHeader
+description       = Test the notnull() function.  This returns 0 for an undefined variable, an empty string, a blank string, an empty array and an empty hash.
+<n>
+<n>--------
+<n>Test the $PG-> pretty_print(object, html/text/TeX, level) function for pretty printing objects.  When called directly as a subroutine (without $PG->) it uses level=5 and the current displayMode to print the object.
+problemList       = 
```

Comment:
These <n> are causing an issue for me on import?  Is that something specific to my setup?